### ISSUE TITLE
Autosuggest linking

### ISSUE THREAD

**Issue by jtheo314** [moroshko/react-autosuggest#424]
In regards to using React-Autosuggest, how would one be able to make clicking one of the autosuggest options and pressing enter/submit link to a page within the site?

**Comment by moroshko** [comment]
Sorry, the question is not clear. Could you rephrase it?
What do you want to happen when suggestion is clicked?
What do you want to happen when <kbd>Enter</kbd> is pressed?

**Comment by jtheo314** [comment]
When the suggestion is clicked, it already shows up within the form. I'd like it to link to a certain page within the site when enter is pressed, based on what is populated in the form

**Comment by moroshko** [comment]
You can try to provide `inputProps.onKeyDown`, validate that <kbd>Enter</kbd> is pressed, and then navigate.

**Comment by jtheo314** [comment]
https://github.com/jnierendorf/Liberty-Hacks/blob/Jason-branch/src/js/components/Searchbar.js

I'm currently fooling around with this code from your codepen to see how it works exactly before deciding whether I want to implement it in this website or not. I was wondering how I can make it so one of the options suggested, once the user clicks on it and presses enter, can link to one of the other pages on this website. For example, once 'skype' is selected and the user clicks Enter, they will be redirected to the website's Skype page.

Thank you!
